rootfs/etc/profile.d/aws-vault.sh

```diff
@@ -48,6 +48,10 @@ if [ "${AWS_VAULT_ENABLED}" == "true" ]; then
 				--preview "$_preview"
 	}
 
+	function choose_role() {
+		[ -n "${AWS_IAM_ROLE_INTERACTIVE}" ] && echo "$(choose_role_interactive)" || echo "${AWS_DEFAULT_PROFILE}"
```

Comment:
thoughts on calling it `ASSUME_ROLE_INTERACTIVE`?

Reply:
since it relates more to the behavior of the `assume-role() ...` function

Reply:
@osterman done